README.md

```diff
@@ -183,6 +183,14 @@ To avoid downloading the OpenThread repo on every new build, set an external pat
 	$ knot ot-path `pwd`
 	```
 
+#### See Log messages through USB
```

Comment:
Warn users that using USB stack the range of connectivity decrease on nrf52840 dongle model.